module/Activity/view/activity/admin/queue.phtml

```diff
@@ -1,3 +1,12 @@
+<h1><?= $this->translate('Update Proposals');?> </h1>
+<?= $this->partial('activity/activity/list.phtml', [
+    'activities' => $updatedActivities,
+    'proposals' => $updateProposals,
+    'admin' => true,
+    'MoreLink' => $moreUpdateProposals ? 'This won\'t happen. Really. If it does, just fix these proposals first.': null,
```

Comment:
Sure. I was just using the template of the pagination and added this. Problem is that actually supporting this means another view has to be made, which I rather don't if it will practically never be used.